roles/openshift_examples/templates.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+EXAMPLES_BASE=$(pwd)/files/examples
```

Comment:
I don't know if this script is always run automatically from the same directory, if not:
```
EXAMPLES_BASE=$(dirname ${BASH_SOURCE[0]})/files/examples
```
Will make this script work correctly even if invoked from different paths:

```
.../openshift-ansible/roles/openshift_examples$ ./templates.sh    # ok
.../openshift-ansible$ roles/openshift_examples/templates.sh      # ok
```